shell.nix

```diff
@@ -9,15 +9,27 @@ in { pkgs ? import (_pkgs.fetchFromGitHub {
 
 with pkgs;
 
-mkShell {
```

Comment:
We should just advance the whole nixpkgs reference.

Reply:
@nshalman, when I do ipxe builds fail. https://github.com/tinkerbell/boots/pull/174#issue-682303973

Reply:
I think that's ipxe/ipxe#210. Might be time to advance iPXE too... @mmlb?

Reply:
Pretty sure we already have that fix applied. Could be that my original patch wasn't a complete fix (as I suspected) and just happened to align right somehow.

Reply:
alternatively, we can do the reverse. Update everything but gcc and pin that.

Reply:
If possible I'd prefer updates like this related to ipxe be a separate PR so that this PR is just related to using the Go built-in embedding.

Reply:
sgtm

Reply:
I guess we untangle all of that later. I approve.